hydroflow_plus/src/stream.rs

```diff
@@ -690,6 +690,42 @@ pub(super) fn deserialize_bincode<T: DeserializeOwned>(tagged: bool) -> Pipeline
 }
 
 impl<'a, T, W, N: Location<'a>> Stream<T, W, NoTick, N> {
+    pub fn decouple_process<P2>(
+        self,
+        other: &Process<P2>,
+    ) -> Stream<'a, T, Unbounded, NoTick, Process<P2>>
+    where
+        N: CanSend<Process<P2>, In<T> = T, Out<T> = T>,
+        T: Clone + Serialize + DeserializeOwned,
+    {
+        match self.location_kind {
```

Comment:
Shouldn't need to do this check, because since `Out<T> = T` we know the source must be a process.